homeassistant/components/home_connect/config_flow.py

```diff
@@ -14,9 +15,14 @@ class OAuth2FlowHandler(
 
     DOMAIN = DOMAIN
 
-    MINOR_VERSION = 2
+    MINOR_VERSION = 3
 
     @property
     def logger(self) -> logging.Logger:
         """Return logger."""
         return logging.getLogger(__name__)
+
+    async def async_oauth_create_entry(self, data: dict) -> ConfigFlowResult:
+        """Create an entry for Electric Kiwi."""
+        await self.async_set_unique_id(DOMAIN)
```

Comment:
I don't think we need this if we only allow one config entry.

We don't need a unique id to do the reauth flow. The flow will look up the correct config entry via the config entry id. The example in the docs does this by default. Just remove the lines that pertains to the unique id from the example.

Reply:
Cool, seems to be working! I will apply those changes about reauth at #136953